packages/client/hmi-client/src/page/project/components/tera-tab-content.vue

```diff
@@ -0,0 +1,161 @@
+		:project="project"
+		@update-tab-name="updateTabName"
+		@create-new-model="createNewModel"
+		@asset-loaded="emit('asset-loaded')"
+		is-editable
+	/>
+	<code-editor
+		v-else-if="assetType === ProjectAssetTypes.CODE"
+		:initial-code="code"
+		@on-model-created="openNewModelFromCode"
+	/>
+	<tera-project-overview
+		v-else-if="assetType === 'overview'"
+		:project="project"
+		@open-workflow="openWorkflow"
+	/>
+	<tera-simulation-workflow v-else-if="assetType === 'workflow'" :project="project" />
+	<!--Add new process/asset views here-->
+	<template v-else-if="assetId && (!isEmpty(tabs) || isDrilldown)">
+		<!--Investigate using component tag since props are similar-->
+		<tera-document
+			v-if="assetType === ProjectAssetTypes.DOCUMENTS"
+			:xdd-uri="getXDDuri(assetId)"
+			:previewLineLimit="10"
+			:project="project"
+			is-editable
+			@open-code="openCode"
+			@asset-loaded="emit('asset-loaded')"
+		/>
+		<tera-dataset
+			v-else-if="assetType === ProjectAssetTypes.DATASETS"
+			:asset-id="assetId"
+			:project="project"
+			is-editable
+			@asset-loaded="emit('asset-loaded')"
+		/>
+		<simulation-plan
+			v-else-if="assetType === ProjectAssetTypes.PLANS"
+			:asset-id="assetId"
+			:project="project"
+			@asset-loaded="emit('asset-loaded')"
+		/>
+		<simulation-run
+			v-else-if="assetType === ProjectAssetTypes.SIMULATION_RUNS"
+			:asset-id="assetId"
+			:project="project"
+			@asset-loaded="emit('asset-loaded')"
+		/>
+	</template>
+	<section v-else class="no-open-tabs">
+		<img src="@assets/svg/seed.svg" alt="Seed" />
+		<p>You can open resources from the resource panel.</p>
+		<Button label="Open project overview" @click="openOverview" />
+	</section>
+</template>
+
+<script setup lang="ts">
+import { ref, watch } from 'vue';
+import * as ProjectService from '@/services/project';
+import { createModel, addModelToProject } from '@/services/model';
+import { ProjectAssetTypes, IProject } from '@/types/Project';
+import { PetriNet } from '@/petrinet/petrinet-service';
+import { useRouter } from 'vue-router';
+import useResourcesStore from '@/stores/resources';
+import { RouteName } from '@/router/routes';
+import { isEmpty, cloneDeep } from 'lodash';
+import { Tab } from '@/types/common';
+import Button from 'primevue/button';
+import TeraDocument from '@/components/documents/tera-document.vue';
+import TeraDataset from '@/components/dataset/tera-dataset.vue';
+import TeraModel from '@/components/models/tera-model.vue';
+import CodeEditor from '@/page/project/components/code-editor.vue';
+import SimulationPlan from '@/page/project/components/Simulation.vue';
+import SimulationRun from '@/temp/SimulationResult3.vue';
+import TeraProjectOverview from '@/page/project/components/tera-project-overview.vue';
+import TeraSimulationWorkflow from '@/components/workflow/tera-simulation-workflow.vue';
+
+const props = defineProps<{
+	project: IProject;
+	assetId?: string;
+	assetType?: ProjectAssetTypes | 'overview' | 'workflow' | '';
```

Comment:
We might want to just add `overview` and `workflow`  and `` to `ProjectAssetTypes`

Reply:
Workflow makes sense. But overview isn't an asset and if we were to add a process like calibrate it wouldn't make sense either. What do people think of having an `AssetType`, a `ProcessType` and a `ProjectPageType` that would hold the `AssetType` + `ProcessType` and any other project page like 'overview'.